### PR TITLE
DCS-1209 Adding removing duplicate service and moving gender over to …

### PR DESCRIPTION
**Pull request by andrewrlee**
…an enum

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/confirmedarrival/ConfirmedArrivalServiceTest.kt [hunk 1]
```diff
@@ -41,7 +41,9 @@ class ConfirmedArrivalServiceTest {
   @Test
   fun `add new confirmed arrival`() {
 
-    every { confirmedArrivalRepository.save(any()) } returns confirmedArrivalDb
+    every {
+      confirmedArrivalRepository.save(any())
+    } returns confirmedArrivalDb
```
Comment (andrewrlee) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/confirmedarrival/ConfirmedArrivalServiceTest.kt:46]: this fails for me locally due to type erasure - don't really understand why it fails for me but not on the build or for others.

File: src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/basm/Model.kt [hunk 2]
```diff
@@ -50,14 +52,19 @@ class Model {
...
-    val key: String,
-    val title: String,
-    val description: String?,
-    val disabled_at: String?,
-    val nomis_code: String?
-  )
+  enum class Gender {
+    MALE, FEMALE;
+
+    companion object {
```
Comment (connormaglynn) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/basm/Model.kt:58]: Ahhh, I was sticking the @JsonCreator in the People object!

File: src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/confirmedarrival/ConfirmedArrivalServiceTest.kt (hunk 1 above)
Comment (connormaglynn) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/confirmedarrival/ConfirmedArrivalServiceTest.kt:46]: Looks okay locally for me, any odd behaviour from Intellij I always go to the 'Invalidate cache', fixes almost every odd local thing for me 😄

**Review by connormaglynn — APPROVED** [review]
lgtm

**Pull request merged**